src/detector/options.py

```diff
@@ -33,24 +34,33 @@ class Options:
 
     def __init__(
         self,
-        proto_dirs_original: str,
-        proto_dirs_update: str,
+        proto_definition_original: str,
+        proto_definition_update: str,
         package_prefixes: str = None,
         human_readable_message: bool = False,
         output_json_path: str = None,
     ):
-        self.proto_dirs_original = self._get_proto_dirs(proto_dirs_original)
-        self.proto_dirs_update = self._get_proto_dirs(proto_dirs_update)
+        self.proto_definition_original = self._get_proto_definition(
+            proto_definition_original
+        )
+        self.proto_definition_update = self._get_proto_definition(
+            proto_definition_update
+        )
         self.package_prefixes = self._get_package_prefixes(package_prefixes)
         self.human_readable_message = human_readable_message
         self.output_json_path = self._get_output_json_path(output_json_path)
 
-    def _get_proto_dirs(self, proto_dirs):
-        proto_dirs_arr = proto_dirs.split(",")
-        for directory in proto_dirs_arr:
+    def _get_proto_definition(self, arg):
+        # Return an array of the proto directories or a descriptor set file path.
+        args = arg.split(",")
+        if os.path.isfile(args[0]):
```

Comment:
args[0] would always exist, since from the `click` module, we configure the `proto_definition_original` and `proto_definition_update ` as argument, not an option, so users have to pass in the path, else the command line will throw error.